terraso_backend/apps/core/models/groups.py

```diff
@@ -153,6 +153,9 @@ def managers_only(self):
     def approved_only(self):
         return self.filter(membership_status=Membership.APPROVED)
 
+    def approved_only(self):
+        return self.filter(membership_status=Membership.APPROVED)
+
```

Comment:
```suggestion
```

Seems to be duplicated from above code

Reply:
Weird! Must have been duplicated in a rebase.